.github/workflows/release.yml

```
@@ -21,6 +21,12 @@ jobs:
       - name : Get release version
         id: get_version
         run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})
+      - name: Generate SBOM
+        uses: CycloneDX/gh-gomod-generate-sbom@v0.2.0
+        with:
+          json: true
+          output: bom.json
+          version: latest
```

Shall the licenses also be included? 
```suggestion
          resolve-licenses: true
          version: latest
```

---

Sure, thanks for the suggestion.